libraries/AP_HAL_ChibiOS/hwdef/MatekF405-Wing/hwdef.dat

```diff
@@ -43,8 +48,7 @@ STM32_VDD 330U
 I2C_ORDER I2C1 I2C2
 
 # order of UARTs
-UART_ORDER OTG1 USART1 USART3 UART4 UART5 USART6
-
+UART_ORDER OTG1 USART3 USART1 EMPTY UART4 UART5 USART6
```

Comment:
humn... this doesn't match the comment you put above?

Reply:
and OTG1 -> Serial0

yep, you are right.  Another change could be to make SerialManager stop reordering the uarts, but then we would need to check all existing boards :-/ and the chance to screw it up is high.

Reply:
Hah, I did think of 'fixing' SerialManager but then thought of the chaos that could cause and quickly thought the better of it :)